src/Admin/Startup.cs

```diff
@@ -95,6 +96,11 @@ public void ConfigureServices(IServiceCollection services)
         });
         services.Configure<RouteOptions>(options => options.LowercaseUrls = true);
 
+        services.Configure<RazorViewEngineOptions>(o =>
+         {
+             o.ViewLocationFormats.Add("/Auth/Views/{1}/{0}.cshtml");
```

Comment:
Point of interest: MVC uses file based routing, so this piece of code tells it where to look for the missing views.